### PR TITLE
Create makefile

### PR DESCRIPTION
**Pull request by SpaceGold**
Hi @naupaka @mhaliz @melt6457, 

My makefile branch successfully builds upon all of our previous work, including the interesting fourth graph that Dr. Z helped me create. That thing shows definitively that people have been far more active on Apple Maps than before covid!

I did note, though, that there is a warning in the section where I make that box plot (attached). I'll look into it next time I sit down to code. 

Thanks for your support!

Rmd warning issue

### CODE REVIEW

File: code/process_sequences.sh [hunk 1]
```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+set -euo pipefail
```
Comment (mhaliz) [inline code/process_sequences.sh:3]: Per specification 2, you should refactor this bash script to take the additional parameter 'ALL' as a second command line parameter. When you run the script with ALL, the script will output everything as you had it originally.

**Review by mhaliz — CHANGES_REQUESTED** [review]
Hi @SpaceGold ,
One quick fix in process_sequences.sh, adding the `ALL` command line parameter.
Other than that, everything looks good!

File: code/process_sequences.sh (hunk 1 above)
Comment (SpaceGold) [reply code/process_sequences.sh:3]: @mhaliz Thanks for the feedback. I've done some digging, but I'm still not sure how to implement this. Would you suggest adding the `@$` parameter? Not sure which line either.

Comment (naupaka) [reply code/process_sequences.sh:3]: @SpaceGold use a nested `if` or a compound logical test, eg something like: this == that & thing2 == pqr

File: code/process_sequences.sh [hunk 2]
```diff
@@ -20,10 +20,14 @@ then
...
-if [ $# -gt 1 ]
+
+# Check for ALL
+if [ "$#" -eq 2 ]
 then
-  echo "Too many parameters passed. Supply 1 fasta.gz file."
-  exit 1
+  if [ $2 == 'ALL' ]
+  then
+    echo "ALL file detected."
```
Comment (SpaceGold) [inline code/process_sequences.sh]: @naupaka @mhaliz @melt6457 Thanks for the detailed feedback. I've added logic, but I'm still unsure of what content goes inside the nested if statements in order to make all the previous output display properly(?).

Comment (naupaka) [reply code/process_sequences.sh]: ALL is not a file. If there is ALL, then do all the things. If there is not ALL, then only do the one thing.

Comment (naupaka) [reply code/process_sequences.sh]: Sounds kind of like a Zen koan.

**Comment by naupaka** [comment]
@SpaceGold you need to get this wrapped up so I can grade it. Hopefully by tomorrow late morning?

**Comment by SpaceGold** [comment]
@naupaka @mhaliz Thanks for your patience! `process_sequences.sh` now meets specifications. Note that when passed `"ALL"`,  it saves additional info to the output file, making the output incompatible with the current build of `Analysis.Rmd`. Without passing `"ALL"`, `Analysis.Rmd` is as smooth and complete as ever.

**Pull request closed without merging**